server/routes/v1/nodes.js

```diff
@@ -29,6 +23,12 @@ const semverCompare = (a = '', b = '') => {
   return 0;
 };
 
+const ledgerCompare = (a = 0, b = 0) => {
+  const aLedger = a.validated_ledger ? a.validated_ledger.ledger_index : 0;
+  const bLedger = b.validated_ledger ? b.validated_ledger.ledger_index : 0;
+  return bLedger === aLedger ? semverCompare(b.version, a.version) : bLedger - aLedger;
+};
+
```

Comment:
why did this have to move?

Reply:
because `ledgerCompare` uses `semverCompare`, and the new linter rules mean that the method must be defined before it can be used